bigquery/src/main/java/org/apache/zeppelin/bigquery/BigQueryInterpreter.java

```diff
@@ -89,7 +89,7 @@
   static final String PROJECT_ID = "zeppelin.bigquery.project_id";
   static final String WAIT_TIME = "zeppelin.bigquery.wait_time";
   static final String MAX_ROWS = "zeppelin.bigquery.max_no_of_rows";
-  static final String LEGACY_SQL = "zeppelin.bigquery.use_legacy_sql";
+  static final String SQL_DIALECT = "zeppelin.bigquery.sql_dialect";
```

Comment:
removing a config would break existing users- how about adding the new sql_dialect but also keep the use_legacy_sql one?

Reply:
that https://github.com/apache/zeppelin/pull/2929/files#r182642493 was what I meant.

but if use_legacy_sql is new and never released than I'm good with removing it